### DISCUSSION TITLE
Mermaid support in Markdown

### DISCUSSION THREAD

**Discussion by biswajit-goswami** [freeplane/freeplane#1570] [Help] [answered]
Freeplane supports markdown. In markdown , we can have some sections using Mermaid/PlantUML syntax...some editors will render the Mermaid diagram in the rendered markdown file.. will it be possible to do something similar using Freeplane ?

**Comment by nnako** [comment, marked as answer]
Hi @biswajit-goswami , this is no direct answer to your question but I want to add another "technology" called `plantuml`. In a similar way as Freeplane handles `markdown` other markup renderers might be easily added to Freeplane's features. It should only be a question of how to integrate external Java libraries and how to render their output within the Freeplane editor. I am looking forward to seeing these renderers come to life... ;-)
